java/test/org/openqa/selenium/grid/config/JsonConfigTest.java

```diff
@@ -35,15 +36,61 @@ void shouldUseATableAsASection() {
     assertThat(config.get("cheeses", "selected")).isEqualTo(Optional.of("brie"));
   }
 
+  @Test
+  void shouldContainConfigFromArrayOfTables() {
+    String raw = String.join("", "",
```

Comment:
More human-readable, but susceptible to alignment change from the formatter. Is there a way to exempt sections of code from being reformatted?